packages/api/src/commands/install.ts

```diff
@@ -51,47 +71,29 @@ cmd
       }
     } else if (Fetcher.isAPIRegistryUUID(uri)) {
       identifier = Fetcher.getProjectPrefixFromRegistryUUID(uri);
+    } else if (oas.api?.info?.title) {
+      identifier = uslug(oas.api.info.title, { lower: true });
+
+      // let useInfoAsIdentifier;
+      const { value: confirmation } = await confirmProposedIdentifier(oas.api.info.title, identifier);
+      if (!confirmation) {
+        // If they don't like what we picked from the spec info doc then let's have them tell us
+        // what they want.
+        ({ value: identifier } = await promptForIdentifier(false));
+      }
     } else {
```

Comment:
We should never get to this `else` case because `info.title` is required in an OpenAPI definition but just in case we don't have it for whatever reason, we'll just fallback to asking them what they want -- though codegen will likely fail immediately after this.